src/module_ncio.f90

```diff
@@ -126,12 +181,12 @@ subroutine nccheck(status,halt,fname)
        if (stopit) stop 99
     end if
   end subroutine nccheck
-  !> Get Dimension object given name.
+  !> @brief Get Dimension object given name.
```

Comment:
Do not use the @brief tag for subroutines/functions. For subroutines/functions, doxygen assumes the first sentence of the documentation is the brief description of the subroutine/function.

Unfortunately that only works for subroutines/functions. For files we still need the @brief tag.